stub/secret-stub.json

```diff
@@ -13,6 +13,7 @@
         "botToken": "An access token with the repo scope used for visible GitHub API access, such as comments"
     },
     "meta": {
+        "pocketmineSecret": "Secret for PocketMine Plugin Repository webhook, You will never need this, ignore it.",
```

Comment:
???

Reply:
Key in github webhook (hash checked) so poggit doesnt accept random api additions from anyone

Reply:
Yes I understood that, my question is why it says this:

>  You will never need this, ignore it.

Reply:
the stub isnt in production, only here for people setting it up outside production so they'll never need it